config/webpack.config.js

```diff
@@ -171,7 +168,7 @@ module.exports = function(webpackEnv) {
 
 		output: {
 			// The build folder.
-			path: isEnvProduction ? paths.appBuild : undefined,
+			path: paths.appBuild,
```

Comment:
In dev it was outputting in `./dist` instead of `./assets/dist`; this is why this change has been made.